app/assets/stylesheets/application/modules/_discussion.scss

```diff
@@ -0,0 +1,335 @@
+  position: relative;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 0;
+  .discussion-title {
+    font-weight: bold;
+  }
+}
+
+.discussion-description {
+  margin-top: 0;
+  .status-label {
+    margin-right: 2px;
+    font-size: 18px;
+  }
+}
+
+.discussion-tabs {
+  margin-top: 20px;
+}
+
+.discussion-info {
+  font-size: 18px;
+}
+
+.discussion-initiator-name {
+  font-weight: bold;
+}
+
+.discussion-message-content {
+  padding: 15px;
+}
+
+.discussion-new-message-content {
+  resize: none;
+  border-radius: 0;
+  border-color: transparent;
+  &:focus {
+    border-color: white;
+  }
+}
+
+summary.discussion-summary {
+  display: list-item;
+  font-size: 29px;
+  outline: 0;
+  cursor: pointer;
+}
+
+.discussion-new-message-button {
+  border-radius: 0;
+  margin-top: 15px;
+}
+
+.discussion-message {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: start;
+  padding: 0;
+  border: none;
+  margin-top: 30px;
+  .discussion-message-user-avatar {
+    margin-right: 20px;
+  }
+}
+
+.discussion-message-bubble-container {
+  width: 100%;
+}
+
+.discussion-message-bubble {
+  background-color: white;
+  border: 1px solid $discussion-message-border-color;
+  border-radius: 3px;
+  position: relative;
+  .discussion-message-bubble-header {
+    background-color: $mu-color-highlight-background;
+    height: 40px;
+    &:before {
+      position: absolute;
+      top: calc(20px - #{$discussion-message-arrow-size + 2px} / 2);
+      left: -$discussion-message-arrow-size - 2px;
+      border-width: (1px + $discussion-message-arrow-size / 2) (2px + $discussion-message-arrow-size) (1px + $discussion-message-arrow-size / 2) 0;
+      content: '';
+      border-style: solid;
+      border-color: transparent $discussion-message-border-color;
+    }
+    .discussion-message-bubble-title {
+      padding: 5px 15px;
+      display: block;
+      .message-date {
+        font-size: 15px;
+      }
+      .actions {
+        float: right;
+        i {
+          color: #aaaaaa
+        }
+      }
+    }
+  }
+}
+
+.discussion-actions {
+  display: inline-block;
+  float: right;
+  margin-top: 35px;
+  > form {
+    display: inline;
+  }
+}
+
+.text-default,
+.text-default:hover {
+  color: #798d8f;
+}
+
+$statuses: (
+  closed: ($brand-danger white $brand-danger),
+  opened: (white #333333 #dddddd),
+  solved: ($brand-success white $brand-success),
+  pending_review: ($brand-success white $brand-success)
+);
+
+@each $status, $style in $statuses {
+  $background-color: nth($style, 1);
+  $font-color: nth($style, 2);
+  $border-color: nth($style, 3);
+
+  .btn-discussion-#{$status} {
+    background-color: $background-color;
+    color: $font-color;
+    border-color: $border-color;
+    margin-left: 5px;
+    &:hover {
+      color: $font-color;
+      border-color: darken($border-color, 5%);
+      background-color: darken($background-color, 5%);
+    }
+  }
+}
+
+
```

Comment:
Too much hardcoded colors in this file, please, use available colors or put it into scss variables